### PR TITLE
txt2tags 3.7

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
01:02:22 cp -pR /private/tmp/d20200311-70691-cc3pas/txt2tags-3.7/. /private/tmp/txt2tags-20200311-70691-tf55o8/txt2tags-3.7
01:02:22 chmod -Rf +w /private/tmp/d20200311-70691-cc3pas
01:02:22 Error: An exception occurred within a child process:
01:02:22   Errno::ENOENT: No such file or directory - txt2tags
```

File: Formula/txt2tags.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 class Txt2tags < Formula
   desc "Conversion tool to generating several file formats"
   homepage "https://txt2tags.org/"
-  url "https://github.com/txt2tags/txt2tags/archive/2.6.tar.gz"
```
Comment (chenrui333) [inline Formula/txt2tags.rb]: This has been written in python
https://github.com/txt2tags/txt2tags/tree/v3 (py source code) vs
https://github.com/txt2tags/txt2tags/tree/v2 (binary install)

**Comment by chenrui333** [comment]
Rewrite the formula in python 3.8

File: Formula/txt2tags.rb [hunk 2]
```diff
@@ -1,20 +1,24 @@
 class Txt2tags < Formula
+  include Language::Python::Virtualenv
+
   desc "Conversion tool to generating several file formats"
   homepage "https://txt2tags.org/"
-  url "https://github.com/txt2tags/txt2tags/archive/2.6.tar.gz"
-  sha256 "c87b911ac5cc97dd2fdb6067601cf4fbd0094a065f6b7a593142fa75e3a924c1"
-  revision 1
+  url "https://github.com/txt2tags/txt2tags/archive/3.7.tar.gz"
+  sha256 "ba745fc1aa98996578afadeb325173f7032a40f161dc4f6f7a7aa08848d11d03"
```
Comment (bayandin) [inline Formula/txt2tags.rb]:
```suggestion
  url "https://files.pythonhosted.org/packages/0e/80/dc4215b549ddbe1d1251bc4cd47ad6f4a65e1f9803815997817ff297d22e/txt2tags-3.7.tar.gz"
  sha256 "27969387206d12b4e4a0eb13d0d5dd957d71dbb932451b0dceeab5e3dbb6178a"
```

Comment (chenrui333) [reply Formula/txt2tags.rb]: good point!

**Pull request closed without merging**